motr/idx_dix.c

```diff
@@ -1228,6 +1262,14 @@ M0_INTERNAL int m0__idx_cancel(struct m0_op_idx *oi)
 	return M0_RC(0);
 }
 
+static void dix_set_idx_flags(struct m0_op_idx *oi)
+{
+	if (ENABLE_DTM0)
```

Comment:
may be we can remove ENABLE_DTM0 and set default as 
oi->oi_flags |= M0_OIF_SKIP_LAYOUT; 
this will enable skip layout feature and disable crow. (which is the desired behavior for DTM when it will be enabled).

Reply:
User application (S3) need to set M0_OIF_CROW and M0_OIF_SKIP_LAYOUT index flags as a configurable parameter.
Remove this logic once configuration option is available in S3.   -- It is added as a "todo"